packages/lesswrong/lib/collections/localgroups/schema.ts

```diff
@@ -23,7 +23,7 @@ const formGroups: Partial<Record<string,FormGroup>> = {
   advancedOptions: {
     name: "advancedOptions",
     order: 2,
-    label: "Advanced Options",
+    label: "Advanced options",
```

Comment:
I also updated the capitalization in this form to match Agnes's preference for just capitalizing the first letter of the phrase, as we'll be using that across the site

Reply:
Mrrr. This doesn't rise to the level of asking-for-forum-gating or arguing about it much, but I do think this looks worse and I feel sad about it.

Reply:
@Raemon I think it does rise to the forum-gating level, if this isn't what you all want on LW and we are planning on doing this across EAF. Maybe it's worth discussing in slack so that everyone is on the same page?

Reply:
Sure. (Also, to be clear I don't know that Ruby agrees with me)